README.md

```diff
@@ -16,35 +16,30 @@
+Online reviews have risen as one of the major factors that influence purchase behaviours of today’s consumers. Thanks to sites like Yelp and, as well as social networks like Twitter. Those reviews provide strong word-of-mouth power that can draw or drive away potential customers.
 
-//toDo ranking algorithm (Sara)
+Companies now embrace reviews as part of their marketing strategy. A Search Engine Land report found as much as 83% of marketers believe managing reviews produces good ROI. Managed correctly, online reviews can also support and improve the effectiveness of your social media strategy.
 
-//toDo weight model (Sara)
+The Bodacious Advisor for Cuisine Over N-joyment ranks the restaurants within London based on their online reviews on Twitter. 
 
 ## Architecture Overview
-The B.A.C.O.N pipeline for ranking restaurants is built on top of 6 components: 
+The B.A.C.O.N pipeline for ranking restaurants is built on top of 5 components: 
 
-1. A client interface to access the ranked data;
-2. An API to enables the access to data inside the storage;
-3. A Data Translation Layer for cleaning and normalising data from Yelp Dataset and Twitter List Members;
-4. A Data Process Layer, responsible for getting tweets about restaurants and formatting the data into our standard before store it;
-5. The Data Analytics Layer stands on top of DataIku, where our team of Data Scientists perform sentiment analysis/ranking algorithms and weight models to get the Restaurant Rank;
-6. We also have a Storage system to keep our precious data protected.
+1. An API to enables the access the Ranked Data inside the storage through an HTTP request;
+2. A Data Transform Layer (DTL) for cleaning and normalising data from Yelp Dataset and Twitter List Members;
+3. A Data Process Layer (DPL), responsible for getting tweets about restaurants and formatting the data into our standard before store it;
+4. The Data Analysis Layer (DAL) stands on top of DataIku, where our team of Data Scientists perform sentiment analysis/ranking algorithms and weight models to get the Restaurant Rank;
+5. We also have a Storage system to keep our precious data protected.
 
-![architecture](images/arch.JPG)
+![architecture](images/arch.png)
 
 The communication path through our components can be seen in the following image. 
 
 ![communication](images/communication.png)
 
+The main point of access B.A.C.O.N data is through our API, which communicates directly with the storage to retrieve ranked data. The storage is also reached by the Data Process Layer, responsible for data management and communication with the Data Analysis Layer. In DAL is where the magic occurs, machine learning algorithms specialised in sentiment analysis classification identify our tweets in Good and Bad and apply our weight model to better rank the restaurants. The DTL is responsible for translating the dataset from YELP to our data structure that will be used for training the algorithms in the DAL.
```

Comment:
Where it reads "point of access B.A.C.O.N data" it should read "point of access to B.A.C.O.N. data".

Note also the missing . in "B.A.C.O.N"  This is done in several places. I think it would be appropriate to change them all.

If you changed to "Data Processing Layer " above, do so also in this sentence.

Please change "In DAL is where the magic occurs, " to "The DAL is where the magic occurs, since this is the place where"

Please change "identify our tweets in Good and Bad" to "identify Positive and Negative tweets"